test/remote/gateways/remote_nuvei_test.rb

```diff
@@ -208,4 +210,11 @@ def test_purchase_using_stored_credentials_merchant_installments_cit
     assert_success recurring_response
     assert_match 'SUCCESS', recurring_response.params['status']
   end
+
+  def test_successful_authorize_with_bank_account
+    @options.update(billing_address: address.merge(country: 'US', state: 'MA'))
+    response = @gateway.authorize(1.25, @bank_account, @options)
```

Comment:
❓ Question:

Just out of curiosity => you are doing a transaction for 1.25 cents, no dollars, and it works?

Reply:
yes, the `1.25` is a Nuvei test value for [Successful deposit](https://docs.nuvei.com/documentation/us-and-canada-guides/ach/?highlight=ach#Payment_with_UPO)